### ISSUE TITLE
Jest complains about extra snapshot files

### ISSUE THREAD

**Issue by AsaAyers** [storybooks/storyshots#34]
I'm using Storybook and Jest (`create-react-app` + `getstorybook`). When I run `npm test`, Jest identifies there are snapshot files with no associated tests and asks to remove them.

```
No tests found related to files changed since last commit.
Press `a` to run all tests, or run Jest with `--watchAll`.


Snapshot Summary
 › 2 snapshot files found, press `u` to remove them..
```

**Comment by arunoda** [comment]
May be you could save Jest's snapshots in a different location.

**Comment by roonyh** [comment]
It looks like jest picks up directories named `__snapshots__` where ever they are. `jest-snapshots` at the moment does not provide a way to change this name. I will look into this. Maybe will have to send them a PR.

**Comment by AsaAyers** [comment]
Here are the steps to reproduce the issue.

```
create-react-app storyshots-34
cd storyshots-34
getstorybook
npm i -D @kadira/storyshots
NODE_ENV=test ./node_modules/.bin/storyshots
npm test
```

At this point `npm test` fails every time because it finds the `.storybook/__snapshots__/`. the `NODE_ENV=test` seems to be a fairly new thing from `create-react-app`. I don't know much about it except it needed to be there for this to run.

**Comment by roonyh** [comment]
@AsaAyers Thanks! Error is reproduced and I am working on a solution.

`jest-snapshot` version 16.0.0 does allow a custom location with a custom extension be used for snapshot files. I am updating its version on jest-16 branch.

@arunoda if we change the extension to something other than `.snap` this problem will go away. How to handle cleaning up of remaining `.snap` files? I guess we will just leave them to be deleted by the user.

Also `jest-snapshot` v16 have small changes in the way how it serialize components. So when updated some existing storyshots will break. So I guess we will have to do a major release (2.0.0)?

**Comment by arunoda** [comment]
@roonyh yep. It's time for a major release.
And let's put a note on the README on how to migrate.

May be we can provide a script to move old snapshots into new.
We could also detect them and prompt the user to apply that.

**Comment by AsaAyers** [comment]
When upgrading my own project, it seems reasonable to just 
1. verify my snapshots with the current version
2. upgrade storyshots
3. run the new storyshots and have it save new snapshots

Since I'm not changing my components at the same time, I can be sure I didn't break anything. I don't think it seems worth any extra work to try to convert existing snapshots.

**Comment by roonyh** [comment]
@AsaAyers, Published v2.0.0. Could you try it out?

I added those steps [here](https://github.com/kadirahq/storyshots/blob/master/CHANGELOG.md#migrating-from-1xx) in the change log.

**Comment by AsaAyers** [comment]
Thanks! I just upgraded storyshots in my project and it solved the problem.
